test/integration/scheduler/filters/filters_test.go

```diff
@@ -2042,6 +2043,53 @@ func TestUnschedulablePodBecomesSchedulable(t *testing.T) {
 				return deletePod(cs, "pod-to-be-deleted", ns)
 			},
 		},
+		{
+			name: "pod with pvc has node-affinity to non-existent/illegal nodes",
```

Comment:
hi  @Huang-Wei , in addition, the `test/integration/scheduler/filters/filters_test.go` test is added in this PR to enhance stability

Reply:
Please add this test to the master branch first

Reply:
> Please add this test to the master branch first

done